### PR TITLE
refactor(action,action-bar,action-group,action-pad,alert,block-section,block,button): remove deprecated `intl*` properties

### PR DESCRIPTION
**Pull request by anveshmekala**
BREAKING CHANGE:  Removed deprecated `intl*` properties , use `messageOverrides` property instead.

- action:
   * Removed the property `intlLoading` , use `messsageOverrides.loading` instead.
   * Removed the property `intlIndicator`, use `messageOverrides.indicator` instead.

- action-bar:
  * Removed the property `intlExpand` , use `messsageOverrides.expand` instead.
  * Removed the property `intlCollapse`, use `messageOverrides.collapse` instead.

- action-group:
   * Removed the property `intlMore` , use `messsageOverrides.more` instead.

- action-pad:
   * Removed the property `intlExpand` , use `messsageOverrides.expand` instead.
   * Removed the property `intlCollapse`, use `messageOverrides.collapse` instead.

- alert:
   * Removed the property `intlClose`, use `messageOverrides.close` instead.

- block:
  * Removed the property `intlExpand` , use `messsageOverrides.expand` instead.
  * Removed the property `intlCollapse`, use `messageOverrides.collapse` instead.
  * Removed the property `intlLoading` , use `messsageOverrides.loading` instead.
  * Removed the property `intlOptions`, use `messageOverrides.options` instead.

- block-section:
  * Removed the property `intlExpand` , use `messsageOverrides.expand` instead.
  * Removed the property `intlCollapse`, use `messageOverrides.collapse` instead.

- button:
  * Removed the property `intlLoading` , use `messsageOverrides.loading` instead.

### CODE REVIEW

File: src/components/action-bar/action-bar.tsx [hunk 1]
```diff
@@ -348,7 +332,6 @@ export class ActionBar
       <ExpandToggle
         el={el}
         expanded={expanded}
-        // rename intl properties for 1.0
```
Comment (driskull) [inline src/components/action-bar/action-bar.tsx:351]: can you create an issue to rename these functional component params? post 1.0 :)

File: src/components/action-pad/action-pad.tsx [hunk 2]
```diff
@@ -155,6 +139,8 @@ export class ActionPad
   // --------------------------------------------------------------------------
 
   connectedCallback(): void {
+    connectLocalized(this);
```
Comment (driskull) [inline src/components/action-pad/action-pad.tsx:142]: were these missing?

File: src/components/action/resources.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ export const CSS = {
 
 export const TEXT = {
   loading: "Loading",
```
Comment (driskull) [inline src/components/action/resources.ts:14]: do we still need this text export?

**Review by driskull — APPROVED** [review]
👍

File: src/components/action-pad/action-pad.tsx (hunk 2 above)
Comment (anveshmekala) [reply src/components/action-pad/action-pad.tsx:142]: this is a duplicate. will get this removed.

File: src/components/action/resources.ts (hunk 3 above)
Comment (anveshmekala) [reply src/components/action/resources.ts:14]: nope. removed.

File: src/components/action-bar/action-bar.tsx (hunk 1 above)
Comment (anveshmekala) [reply src/components/action-bar/action-bar.tsx:351]: https://github.com/Esri/calcite-components/issues/6043

File: src/components/action/action.tsx [hunk 4]
```diff
@@ -295,7 +278,6 @@ export class Action
       buttonId,
       messages
     } = this;
-
     const ariaLabel = `${label || text}${indicator ? ` (${messages.indicator})` : ""}`;
```
Comment (anveshmekala) [inline src/components/action/action.tsx:281]: are parantheses in `(${messages.indicator})` intentional ?

Comment (anveshmekala) [reply src/components/action/action.tsx:281]: @driskull

Comment (driskull) [reply src/components/action/action.tsx:281]: Yes, they are intentional.

**Pull request merged**